crates/tauri-runtime-wry/src/lib.rs

```diff
@@ -3096,6 +3127,32 @@ fn handle_user_message<T: UserEvent>(
           WindowMessage::RequestRedraw => {
             window.request_redraw();
           }
+          WindowMessage::SetBadgeCount(_count, _desktop_filename) => {
+            #[cfg(target_os = "ios")]
+            window.set_badge_count(
+              _count.map_or(0, |x| x.clamp(i32::MIN as i64, i32::MAX as i64) as i32),
```

Comment:
why we clamping here?

Reply:
because in iOS we can only pass i32, but other platforms support i64 ranges